src/util/system.h

```diff
@@ -336,6 +326,16 @@ class ArgsManager
      */
     std::string GetArg(const std::string& strArg, const std::string& strDefault) const;
 
+    /**
+     * Return path argument or default value
+     *
+     * It is guaranteed that the returned path has no trailing slashes.
+     *
+     * @param arg Argument to get a path from (e.g., "-datadir", "-blocksdir" or "-walletdir")
+     * @return Lexically normalized path without trailing separators
+     */
+    fs::path GetPathArg(const std::string& arg, const fs::path& default_value={}) const;
```

Comment:
In #24265 I was thinking about choosing `const std::string& arg` vs `std::string arg`.

Considering that (a) the longest `arg` in our codebase is 12 characters long, and (b) modern compilers optimization methods (small string optimization, copy elision etc), I've chosen to pass `arg` by value.

Has my consideration a flaw?

---

style nit: `clang-format-diff.py` insist on:
```suggestion
    fs::path GetPathArg(const std::string& arg, const fs::path& default_value = {}) const;
```